docs/docs/09-developer-tools/01-cli.md

```diff
@@ -2,6 +2,12 @@
 
 `templ` provides a command line interface. Most users will only need to run the `templ generate` command to generate Go code from `*.templ` files.
 
+:::tip Using templ as a Go tool
```

Comment:
This admonition isn't formatted correctly.

https://docusaurus.io/docs/next/markdown-features/admonitions

Did you run the docs website and check that it renders correctly?